content/1_General/Adding_Solution.mdx

```diff
@@ -86,23 +86,28 @@ See [Working With MDX](/general/working-mdx) for additional information.
     ```
 
     Keep file names and solution IDs consistent; namely, USACO IDs should
-    correspond to the URL on
+    correspond to the last digits at the very end of the URL on
     [usaco.org](http://www.usaco.org/index.php?page=viewproblem2&cpid=690).
```

Comment:
```suggestion
    [usaco.org](http://www.usaco.org/index.php?page=viewproblem2&cpid=690). See [Working With MDX](/general/working-mdx#problem-lists) for more examples of IDs.
```
Is this the latest version of the file? the version on the guide doesn't exactly match what you have here. can you merge master into this branch?